CONTRIBUTING.md

```diff
@@ -166,17 +216,22 @@ Fixes: #123
 Signed-off-by: Sebastien Boeuf <sebastien.boeuf@intel.com>
 ```
 
-Correct formatting of the PR patches is verified using the
-[checkcommits](https://github.com/kata-containers/tests/tree/master/cmd/checkcommits)
-tool.
+#### Supplementary patch
 
-Note, that the body of the message should not just be a continuation of the
-subject line, and is not used to extend the subject line beyond its length
-limit. They should stand alone as complete sentence and paragraphs.
+If a PR contains multiple patches, [only one of those patches](#main-patch) needs to specify the "`Fixes #XXX`" comment. Supplementary patches have an identical format to the main patch, but do not need to specify a "`Fixes #XXX`"
+comment.
```

Comment:
I learned so much from this doc, I don't know this item either :sweat_smile: 

:+1: